src/components/AppSidebar/AppSidebar.vue

```diff
@@ -425,16 +472,18 @@ $top-buttons-spacing: 6px;
 				text-overflow: ellipsis;
 			}
 			// main title
-			.app-sidebar-header__title {
+			.app-sidebar-header__maintitle {
 				padding: 0;
+				min-height: 30px;
 				font-size: 20px;
 				line-height: $desc-title-height;
 			}
-			input.app-sidebar-header__title-input {
-				width: 100%;
+			input.app-sidebar-header__maintitle-input {
+				flex: 1 1 auto;
 				margin: 0;
 				padding: $desc-input-padding;
-				font-size: 16px;
+				font-size: 20px;
+				font-weight: bold;
```

Comment:
We rarely use bold on Nextcloud. @jancborchardt :)

Reply:
It is to make the text appear the same way as the `<h2>` is rendered. Otherwise it will look different and jump when editing.